src/test/javascript/spec/component/shared/metis/postings-markdown-editor/postings-markdown-editor.component.spec.ts

```diff
@@ -171,6 +178,53 @@ describe('PostingsMarkdownEditor', () => {
         expect(component.lectureAttachmentReferenceAction).toEqual(new LectureAttachmentReferenceAction(metisService, lectureService));
     });
 
+    it('should have set the correct default commands on init if faq is enabled', () => {
+        jest.spyOn(CourseModel, 'isFaqEnabled').mockReturnValueOnce(true);
+        component.ngOnInit();
+
+        expect(component.defaultActions).toEqual(
+            expect.arrayContaining([
+                expect.any(BoldAction),
+                expect.any(ItalicAction),
+                expect.any(UnderlineAction),
+                expect.any(QuoteAction),
+                expect.any(CodeAction),
+                expect.any(CodeBlockAction),
+                expect.any(EmojiAction),
+                expect.any(UrlAction),
```

Comment:
I think in these tests it would be nicer to explicitly check for the Actions which are different. That would also get rid of some duplicated code, as the actions which are always present can be tested in a method, which multiple tests call.

Reply:
Adressed